internal/zinc-compile-core/src/main/scala/sbt/internal/inc/CompilerCache.scala

```diff
@@ -57,6 +58,17 @@ final class CompilerCache(val maxInstances: Int) extends GlobalsCache {
           override def commandArguments(sources: Array[File]): Array[String] = {
             compiler.commandArguments(sources)
           }
+          @silent
```

Comment:
What's being silenced?

Reply:
Deprecation warning that I'm using `run` that takes `Array[File]`.